bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -509,7 +533,31 @@ func (r *KubeadmConfigReconciler) joinWorker(ctx context.Context, scope *Scope)
 			UseExperimentalRetry: scope.Config.Spec.UseExperimentalRetryJoin,
 		},
 		JoinConfiguration: joinData,
-	})
+	}
+
+	var cloudJoinData []byte
+
+	switch scope.Config.Spec.Format {
+	case bootstrapv1.Ignition:
+		ign, warnings, err := ignition.NewNode(&ignition.NodeInput{
+			NodeInput: nodeInput,
+			Ignition:  scope.Config.Spec.Ignition,
+		})
+
+		if err != nil {
+			scope.Error(err, "Failed to Ignition user data for bootstrap control plane")
```

Comment:
```suggestion
			scope.Error(err, "Failed to generate Ignition user data for bootstrap worker")
```

Reply:
Maybe `Failed to generate Ignition user data for the worker node` to be consistent with `Creating BootstrapData for the worker node` above, even though the error below this one is not consistent?

Reply:
@jzhoucliqr I'll mark this as resolved, I improved the message. Let me know if you have some further comments.